app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Models/Place.java

```diff
@@ -52,14 +54,18 @@ public String getName() {
 
     public String getPlaceID() {
         return placeID;
-
     }
+
     public String getType() {
         return type;
     }
 
     @Override
     public String toString() {
-        return latitude + "," + longitude;
+        if (latitude != null) {
+            return getLatitude() + "," + getLongitude();
+        } else {
+            return getPlaceID();
+        }
```

Comment:
I think this might be better if you had "return (latitude != null) ? getLatitude() + "," + getLongitude() : getPlaceID();"
My reasoning is that it has only one "return" so to speak. It makes it easier to determine where the function ends at a glance.

Reply:
Thanks! Changed as per your request.